src/Polytopes/helpers.jl

```diff
@@ -11,6 +13,9 @@ function Polymake.Matrix{Polymake.Rational}(x::Union{Oscar.fmpq_mat,AbstractMatr
     return res
 end
 
+_isempty(x::Pair{<:Union{Oscar.MatElem, AbstractMatrix}, Any}) = isempty(x[1])
+_isempty(x) = isempty(x)
```

Comment:
Maybe we should make this function name more specific, to avoid clashes with anyone else who might try to define `_isempty`.
```suggestion
_isempty_halfspace(x::Pair{<:Union{Oscar.MatElem, AbstractMatrix}, Any}) = isempty(x[1])
_isempty_halfspace(x) = isempty(x)
```